uplink/src/base/serializer.rs

```diff
@@ -363,12 +454,6 @@ impl Serializer {
     /// [slow mode]: Serializer::slow
     /// [crash mode]: Serializer::crash
     pub async fn start(mut self) -> Result<(), Error> {
-        if self.storage.is_none() {
```

Comment:
With latest changes, for system with No Persistence, the state change will be: [`EventloopReady`] -> `catchup()` (No Persistence) -> [`Normal`] -> `normal()`

And in the case of network slowing down:
[`SlowEventloop(publish)`] -> `slow(publish)` from where we can branch in two ways
1. Network is back up, so [`Normal`] -> `normal()` (All data received in between is lost and is [print to error log](https://github.com/bytebeamio/uplink/pull/29/files#diff-fb2332567a4ea31637aad3f6525bfa09d5a2c3dd4f7b414e9bbb38c1a188e393R252)).
2. Network dies out, [`EventloopCrash(publish)`] -> `crash(publish)`; `Serializer` errors out due to No Persistence, thus ending process.